src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email).isPresent()) {
+            User user = userService.findByEmail(email).get();
```

Comment:
let's save this `userService.findByEmail(email)` into a local variable and reuse it